robottelo/host_helpers/host_mixins.py

```diff
@@ -0,0 +1,32 @@
+# Host helpers related to File Downloading and operations on host objects
+# Replace os with tmpfile library
+import os
+
+from robottelo.errors import DownloadFileError
+
+
+class IOHelpers:
+    def download_file(self, file_url, local_path=None, file_name=None):
```

Comment:
This looks like overlap with this function:

https://github.com/SatelliteQE/robottelo/blob/87a748f761637ce44bfc979818701c5d4867f553/robottelo/hosts.py#L538

Starting to get concerned about the subclassing being done here, and the potential for duplicate definitions and namespace collisions.

Reply:
That's a `put` operation whereas https://github.com/SatelliteQE/robottelo/blob/87a748f761637ce44bfc979818701c5d4867f553/robottelo/hosts.py#L534 is the right method.

I wonder why its just for ContentHost and not in `Host` base class. But even thats the case, we can remove `download_file` and start using `get` altogether.

Reply:
Done !

Reply:
Well, both put and get are both already defined there, I was just linking one. 

Going to push a commit to your PR to fix the broken import since this file/class was removed.